kadi/commands/tests/test_states.py

```diff
@@ -1211,17 +1211,23 @@ def test_get_pitch_from_mid_maneuver():
     """
     start = '2019:039:14:16:25.002'  # Mid-maneuver
     stop = '2019:039:16:00:00.000'
-    exp = ['      datestart              datestop           pitch     pcad_mode',
+    exp = Table.read(
+          ['      datestart              datestop           pitch     pcad_mode',
```

Comment:
I think this change violates flake8 because I tried to leave the columns where they were.  Up to @taldcroft

Reply:
Better to leave the original `exp = [' .... ']` code and add `exp = Table.read(exp, format='ascii')`.  It's preferred not to have giant args in a function call.